webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -123,6 +123,7 @@ export const GlideTable: React.FC<Props> = ({
 
   const getRowThemeOverride = React.useCallback(
     (row: number): Partial<Theme> | undefined => {
+      if (!data[row]) return;
```

Comment:
Somehow `row` can be greater than number of items in `data`, add this to avoid error in Loadable